lib/fauxpaas/cli/main.rb

```diff
@@ -15,13 +17,15 @@ class Main < Thor
       desc "deploy <instance>",
         "Deploys the instance's source; by default deploys master. Use --reference to deploy a specific revision"
       def deploy(instance_name)
+        Fauxpaas.config = configuration(options)
```

Comment:
This is here because the CLI modifies the configuration with options it gets from the command line. Ideally there would be some sort of hook after option parsing that we could attach this to in Thor, but I could not find one.